public/static/js/task.js

```diff
@@ -400,6 +400,25 @@ mciModule.controller('TaskHistoryDrawerCtrl', function($scope, $window, $locatio
         return orderedTestStatuses.indexOf(task.test_result.status);
       }
 
+      // filter tests in a task based on their display name
+      $scope.filterTests = function() {
+        if($scope.task.searchField != "")
+        {
```

Comment:
Nit: let's keep the opening brace on the line before

Reply:
Fixed